### PR TITLE
fix: add enterprise_project_id to rocketmq instance

### PR DESCRIPTION
**Pull request by houpeng80**
add enterprise_project_id to rocketmq instance
**What this PR does / why we need it**:
  
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  add enterprise_project_id to rocketmq instance
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms/' TESTARGS='-run TestAccDmsRocketMQInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccDmsRocketMQInstance_basic
=== PAUSE TestAccDmsRocketMQInstance_basic
=== CONT  TestAccDmsRocketMQInstance_basic
--- PASS: TestAccDmsRocketMQInstance_basic (718.46s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       718.511s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func TestAccDmsRocketMQInstance_basic(t *testing.T) {
 				),
 			},
 			{
-				Config: testDmsRocketMQInstance_update(rName, updateName),
+				Config: testDmsRocketMQInstance_update(updateName),
 				Check: resource.ComposeTestCheckFunc(
 					rc.CheckResourceExists(),
 					resource.TestCheckResourceAttr(resourceName, "name", updateName),
```
Comment (hmbig2) [inline huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go:78]: pls check the new attr: enterprise_project_id

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go [hunk 2]
```diff
@@ -133,17 +133,17 @@ resource "huaweicloud_dms_rocketmq_instance" "test" {
   broker_num        = 1
   enable_acl        = true
 }
-`, testAccDmsRocketmqInstance_Base(rName), updateName)
+`, testAccDmsRocketmqInstance_Base(name), name)
 }
 
-func testDmsRocketMQInstance_update(rName, name string) string {
+func testDmsRocketMQInstance_update(name string) string {
```
Comment (hmbig2) [inline huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go:141]: the config of `testDmsRocketMQInstance_update` and `testDmsRocketMQInstance_basic` is same. 
remove one?

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go [hunk 3]
```diff
@@ -326,7 +327,7 @@ func resourceDmsRocketMQInstanceCreate(ctx context.Context, d *schema.ResourceDa
 
 	d.SetId(id.(string))
 
-	if _, ok = d.GetOk("cross_vpc_accesses"); ok {
+	if _, ok := d.GetOk("cross_vpc_accesses"); ok {
 		if err = updateCrossVpcAccesses(createRocketmqInstanceClient, d); err != nil {
 			return diag.Errorf("Failed to update default advertised IP: %v", err)
```
Comment (hmbig2) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go]:
```suggestion
			return diag.Errorf("failed to update default advertised IP: %v", err)
```

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go:78]: done

Comment (houpeng80) [reply huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go:78]: done

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_instance_test.go:141]: the param of enable_acl is different

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go (hunk 3 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go]: done

**Pull request merged**